### PR TITLE
fix(feedback): bubble up feedback capture to objc layer

### PR DESCRIPTION
**Pull request by armcknight**
fixes the last build issue in #4652; #skip-changelog; for https://github.com/getsentry/sentry-cocoa/issues/4272

### CODE REVIEW

File: Sources/Swift/Integrations/UserFeedback/SentryFeedback.swift [hunk 1]
```diff
@@ -32,7 +32,7 @@ class SentryFeedback: NSObject, SentrySerializable {
         super.init()
     }
     
-    func serialize() -> [String: Any] {
+    public func serialize() -> [String: Any] {
```
Comment (brustolin) [inline Sources/Swift/Integrations/UserFeedback/SentryFeedback.swift:35]: Curious why do you set this function public if the class is not public?

File: Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackWidget.swift [hunk 2]
```diff
@@ -7,6 +7,10 @@ import UIKit
 
 var displayingForm = false
 
+protocol SentryUserFeedbackWidgetDelegate: NSObjectProtocol {
```
Comment (brustolin) [inline Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackWidget.swift:10]: l: If this is a `NSObjectProtocol` should it also have `@objc`?

**Review by brustolin — APPROVED** [review]
A couple of comments but LGTM.

File: Sources/Swift/Integrations/UserFeedback/SentryFeedback.swift (hunk 1 above)
Comment (armcknight) [reply Sources/Swift/Integrations/UserFeedback/SentryFeedback.swift:35]: oh, leftover from when i thought the class needed to be public, didn't remove this one again 👍🏻

File: Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackWidget.swift (hunk 2 above)
Comment (armcknight) [reply Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackWidget.swift:10]: correct, it doesn't need it, it's implied

**Pull request merged**